OSSM-8934: Conceptual topic for cluster-wide migration (#88982)
openshift/openshift-docs: modules/ossm-cluster-wide-migration-methods.adoc

// Module included in the following assemblies:

// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

:_mod-docs-content-type: CONCEPT
[id="ossm-cluster-wide-migration-methods_{context}"]
= Cluster-wide migration methods

> Migrating the control plane from {SMProductName} {SMv2Version} to {SMProduct} 3.0 can be performed by using one of the following methods:

Reviewer: ```suggestion
You can migrate the control plane from {SMProductName} {SMv2Version} to {SMProduct} 3.0 by using one of the following methods:
```
- To better engage the reader, IBM Style advises writing in second person: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#second-person

Author: done

* Applying the `istio.io/rev` label
* Applying the `istio-injection=enabled` label

> * Performing a simple migration using the `istio-injection=enabled` label

Reviewer: ```suggestion
* Performing a simple migration by using the `istio-injection=enabled` label
```
- Avoid ambiguity by writing either "by using" or "that uses" instead of "using": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#using

Author: done
